src/tetris_application.hpp

```diff
@@ -49,6 +49,7 @@ struct TetrisApplication : public Application {
     }
 
     void update(double delta_time) override {
+        (void)delta_time;
         m_game.update();
     }
```

Comment:
Let's use `[[maybe_unused]]` or remove the name of the parameter all together.
```suggestion
    void update([[maybe_unused]] double delta_time) override {
        m_game.update();
    }
```

Reply:
Also to override the update function (that definitely need a delta_time) we need the parameter :sweat_smile:

Reply:
So the solution would be: `void update(double) override`